src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/QueueClientsConfiguration.java

```diff
@@ -3,24 +3,34 @@
 import com.microsoft.azure.servicebus.ClientFactory;
 import com.microsoft.azure.servicebus.IMessageReceiver;
 import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
 import com.microsoft.azure.servicebus.primitives.ServiceBusException;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 
 @Configuration
+@EnableConfigurationProperties(QueueConfigurationProperties.class)
 @Profile("!functional & !integration")
 public class QueueClientsConfiguration {
 
+    private final QueueConfigurationProperties queueProperties;
+
+    public QueueClientsConfiguration(QueueConfigurationProperties queueProperties) {
+        this.queueProperties = queueProperties;
+    }
```

Comment:
can this be injected to bean getter instead?

Reply:
at first i did try to construct bean but i think i got some other problem giving me false assumptions. i try revert that back too and see how it goes